packages/@glimmer/interfaces/lib/components/component-manager.d.ts

```diff
@@ -21,6 +22,25 @@ export interface ComponentManager<
   getSelf(state: ComponentInstanceState): Reference;
   getDestroyable(state: ComponentInstanceState): Option<Destroyable>;
   getDebugName(state: ComponentDefinitionState): string;
+  getDebugRenderTreeName?(state: ComponentDefinitionState): string;
+}
+
+interface CustomRenderNode extends RenderNode {
+  bucket: object;
+}
+
+export interface WithCustomDebugRenderTree<
+  ComponentInstanceState = unknown,
+  ComponentDefinitionState = unknown
+> extends ComponentManager<ComponentInstanceState, ComponentDefinitionState> {
+  // APIs for hooking into the debug render tree, used by components that
+  // represent multiple logical components. Specifically, {{mount}} and {{outlet}}
```

Comment:
What about `{{component`? (no need to update the message for that, but I'm curious if that falls into the same bucket)

Reply:
It does not currently, but it could in the future. It couldn't have done this previously since `{{component}}` is implemented in the VM, so it didn't have access